Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -75,6 +76,11 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Location manager that is used to track accuracy and status authorization changes.
+     */
+    var locationManager = CLLocationManager()
```

Comment:
And why can't we use the existing location provider here? Or to request changes in Maps SDK is there are missing methods/fields.

Reply:
`LocationProviderDelegate` is not a reliable source as it can be overwritten by the end user. We're actually doing this in case of CarPlay as `PassiveLocationProvider` is always enabled there.